Assets/SEE/Game/InteractionDecorator.cs

```diff
@@ -40,10 +40,7 @@ public static void PrepareForInteraction(GameObject gameObject)
             gameObject.AddComponentIfNecessary<InteractableObject>();
             // The following additions of components must come after the addition of InteractableObject
             // because they require the presence of an InteractableObject.
-            if (SceneSettings.InputType == PlayerInputType.VRPlayer)
-            {
-                gameObject.AddComponentIfNecessary<XRSimpleInteractable>().colliders.Add(gameObject.GetComponent<BoxCollider>());
-            }
+            gameObject.AddComponentIfNecessary<XRSimpleInteractable>().colliders.Add(gameObject.GetComponent<BoxCollider>());
```

Comment:
Well, I thought this script would compile when starting the game, but correct me if im wrong, it only executes when the graph gets drawn in the editor. In this case, there is no VR-Platform or Desktop-Platform, because we are not in game.

Reply:
Yes, this part is called whenever a new component (node/edge) is added to the scene. While this usually happens in the editor, it can also happen in-game, since there's also the option to create code-cities while the game is running.

Reply:
Yes, but we would get different results in editor/in-game, which would not be very nice I think.

Reply:
Ah, I think I understand what you mean now, sorry. To clarify: When a city is created in the editor, the graph drawing actually happens twice: Once in the editor, and again when SEE is started normally (because these game objects wouldn't serialize correctly, at least I think that was the reason). So I think we should only add this component if the current player type is the VRPlayer—in the editor, this won't matter since the interactable is never truly active, and in play mode this check will happen again and should work correctly.

Reply:
I tested it again, but it does not work :/ Maybe this script is just run once in the editor or if you load a city in-game, but not at the start of the game..

Reply:
Hm, that's weird, I could've sworn the graph is drawn again at the start of the game... In that case, as long as the XR interactable component doesn't cause any issues in Desktop mode, you can just leave it as-is (without the if condition).

Reply:
Maybe it is drawn again, but with the attributes that got added when it got first drawn in the editor.

Reply:
Yes, that's possible. Anyway, since it works in desktop mode we can just add the component unconditionally.